Codeinwp/neve
# fix: closing the sidebar menu via button animation [closes #1000]

### Summary
Fixes sidebar animation mentioned in #1000.


### Will affect visual aspect of the product

NO

### Test instructions


- Check if both clicking the close button and outside of the sidebar yields the same animation


Closes #1000.
